infra/src/secrets.ts

```diff
@@ -77,7 +100,11 @@ yargs(hideBin(process.argv))
 
       const { Parameter } = await ssm.getParameter(parameterInput).promise()
       if (Parameter) {
-        return logger.info(Parameter.Value)
+        if (Parameter.Value && Parameter.Value.length > 0) {
+          console.log(Parameter.Value)
```

Comment:
Do we want to console log what comes out of parameter store?

Reply:
In this case yes, If we use the logger, everything is always put into STD out which means that this export `a=$(node secrets get "$PGPASSWORD_KEY")` will ALWAYS print the output. Using console.log changes this behavior and will not print the secret

Reply:
I dont follow. We need the value for the `create-db.sh` script

Reply:
thought this was something new was just checking the code

Reply:
Yes, the values is set in env in both cases. But using `console.log()` it is NOT printed in the console. If we use the logger, the env value is set AND it is always printed in plaintext to the console

Reply:
cool thanks for pointing out. I thought this was just debug log.